src/screens/app/screens/AddTask/screens/ChooseCategory.tsx

```diff
@@ -9,29 +9,29 @@ import { FormLayout } from 'library/layouts/FormLayout'
 import { Label } from 'library/components/Label'
 import { CardButton } from 'library/components/CardButton'
 
-import { wait } from 'res/utils'
 import { useT } from 'res/i18n'
 
 export const ChooseCategoryScreen = () => {
   const { t } = useT()
-  const [selectedCategoryId, setSelectedCategoryId] = useState('')
+  const navigation = useNavigation()
+  const [selectedId, setSelectedId] = useState('')
```

Comment:
i sugest using useState<string | null>(null) null is a better value when unselected category